### ISSUE TITLE
Video/header doesn't load

### ISSUE THREAD

**Issue by duemunk** [bemyeyes/bemyeyes-website#31]
Checked on Safari, Chrome and Firefox. All fails.

![screen shot 2015-01-29 at 10 00 50](https://cloud.githubusercontent.com/assets/489138/5954914/d0fe81ce-a79d-11e4-9d6e-49a78b989d88.png)

**Comment by ksmandersen** [comment]
Could you try to open the Developer Concole and post the output here?

**Comment by duemunk** [comment]
Console output
```
[Error] Failed to load resource: the server responded with a status of 403 (Forbidden) (appstore-hover_2x.png, line 0)
[Error] Failed to load resource: the server responded with a status of 403 (Forbidden) (email_2x.png, line 0)
[Error] Failed to load resource: the server responded with a status of 403 (Forbidden) (play-hover_2x.png, line 0)
[Error] Failed to load resource: the server responded with a status of 403 (Forbidden) (appstore_2x.png, line 0)
[Error] Failed to load resource: the server responded with a status of 403 (Forbidden) (twitter_2x.png, line 0)
[Error] Failed to load resource: the server responded with a status of 403 (Forbidden) (header_2x.jpg, line 0)
[Error] Failed to load resource: the server responded with a status of 403 (Forbidden) (play_2x.png, line 0)
[Error] Failed to load resource: the server responded with a status of 403 (Forbidden) (facebook_2x.png, line 0)
[Warning] The "fb-root" div has not been created, auto-creating (sdk.js, line 61)
[Log] API Stat response (main.js, line 8)
[Log] Object (main.js, line 11)

[Error] SyntaxError: Unexpected EOF
	(anonymous function) (_squish.js, line 3)
[Error] Failed to load resource: the server responded with a status of 403 (Forbidden) (play-hover_2x.png, line 0)
```

**Comment by ksmandersen** [comment]
Oh, so this is a retina issue again. But that doesn't explain the video

**Comment by duemunk** [comment]
The video does play. Sorry about that confusion.

**Comment by ksmandersen** [comment]
Fixed in f2fa285
